client/components/select-dropdown/item.jsx

```diff
@@ -63,6 +64,7 @@ class SelectDropdownItem extends Component {
 							<Count count={ this.props.count } compact={ this.props.compactCount } />
 						</span>
 					) }
+					{ this.props.extra }
```

Comment:
What happens if an item has both `extra` and `count`?

Reply:
In cases where we have both, only the count would be shown.